app/components/form_components/autocomplete/sort/clean.js

```diff
@@ -1,4 +1,11 @@
-const clean = (word) => word.trim()
+const stopWords = ['the', 'of', 'in', 'and', 'at', '&']
+
+const removeStopWords = (text) => {
+  const regex = new RegExp(stopWords.map(word => `\\s+${word}\\s+`).join('|'), 'gi')
```

Comment:
After exploring some of the ideas suggested, this regex solution emerged. The idea is:

1. stops words at the beginning should be ignored because they could be the start of a non-stop word, e.g. "the" => "theatre". There is also the possibility that the user wants to begin the search with "the university" which is valid.

2. If the stop word lies somewhere within the text and separated by a space (both sides), then it's removed. e.g. "the university of norwich" => "the university norwich"

3. If the stop word is at the end of the text with no space, then the user could be typing out a non-stop word, e.g. "university and" => "university andrews".